INSTALL.md

```diff
@@ -30,53 +30,12 @@ Connect an Ethernet cable or set-up wifi. Power as normal via it's power input.
 
 ## Steps for all Pis
 
-Install chromium:
-
-    sudo apt-get update
-    sudo apt-get install rpi-chromium-mods
-
-## Install node
-
-
-Use [radiodan provisioning](https://github.com/radiodan/provision) to install node and also to make everything on port 80 forward to port 5000. 
-
-Once you've followed the setup instructions there:
-
-    sudo ./provision node iptables
```

Comment:
The `iptables` step [forwards port 80 to 5000](https://github.com/radiodan/provision/tree/master/steps/iptables) so you can visit `http://raspberrypi.local` to access the apps. One for later defo.

Reply:
I will also add this now